style.css

```diff
@@ -101,4 +101,27 @@ button {
     text-transform: uppercase;
     background: var(--accent-color);
     color: #fff;
+}
+
+
+/*
+**Desktop draft
+*/
+
+@media only screen and (min-width: 961px) {
+    section {
+        margin: 3rem;
+        padding: 3rem;
+        font-size: .2rem;
```

Comment:
This does nothing since we use rem for in the text size everywhere.

Reply:
Yeah, wasn't the best i can do. I think your solution next bellow will do the work.